charts/zipkin/templates/deployment.yaml

```diff
@@ -79,6 +89,7 @@ spec:
             periodSeconds: 5
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
+        {{- include "zipkin.extraContainers" . | nindent 6 }}
```

Comment:
I don't really know why entirely, but you can't inline the expression. it causes an obscure key not found. I poked around and found otel did the same thing to dodge this, but there wasn't an explicit comment why.